### DISCUSSION TITLE
MINIO files on server kept as encrypted/compressed?

### DISCUSSION THREAD

**Discussion by whitepiratebaku** [minio/minio#18065] [Q&A] [answered]
we have moved from RELEASE.2022-10-15T19-57-03Z to latest version, we see that files kept as xl.meta unlike clear text which older minio version was keeping, this is something configurable? like  to make latest minio also keep  files as clear text?

**Comment by shtripat** [comment]
MinIO uses Erasure Coding to shard data on disks. Objects wont be kept plain on disk as is.

**Comment by klauspost** [comment, marked as answer]
No. MinIO has unified the backend so files are stored in the same format across the board.

This change was done more than a year ago.

https://blog.min.io/deprecation-of-the-minio-gateway/

https://blog.min.io/minio-gateway-migration/
